aiida/repository/backend/disk_object_store.py

```diff
@@ -65,16 +65,11 @@ def _put_object_from_filelike(self, handle: typing.BinaryIO) -> str:
         """
         return self.container.add_object(handle.read())
```

Comment:
```suggestion
        return self.container.add_streamed_object(handle)
```

Files should be read/written in chunks, so that very large files never fill the RAM